### PR TITLE
[WIP] Transit gateway attachment

### PR DESCRIPTION
**Pull request by gen2fish**
##### SUMMARY
New modules to describe and manage transit gateway attachements

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
module: ec2_transit_gateway_attachment_info
short_description: Gather information about ec2 transit gateway attachments in AWS

module: ec2_transit_gateway_attachment
short_description: Manage ec2 transit gateway attachments in AWS

### CODE REVIEW

File: tests/integration/targets/lambda/defaults/main.yml [hunk 1]
```diff
@@ -4,3 +4,4 @@
 # we hash the resource_prefix to get a shorter, unique string
 lambda_function_name: '{{ tiny_prefix }}'
 lambda_role_name: 'ansible-test-{{ tiny_prefix }}-lambda'
+tiny_prefix: "d7er9e"
```
Comment (jillr) [inline tests/integration/targets/lambda/defaults/main.yml:7]: tiny_prefix is a special variable provided by ansible-test and does not need to be defined

Comment (gen2fish) [reply tests/integration/targets/lambda/defaults/main.yml:7]: opps! I didn't think that made it in there! My bad

**Comment by tremble** [comment]
Superceeded by #1004

**Comment by tremble** [comment]
@gen2fish Thanks for your submission here, unfortunately it looks like the PR stalled out.

**Pull request closed without merging**